src/pudl/output/ferc1.py

```diff
@@ -1082,11 +1137,10 @@ def exploded_pks(self) -> list[str]:
             )
         # Some xbrl_factoid names are the same in more than one table, so we also add
         # table_name here.
-        pks = (
-            pudl.helpers.dedupe_n_flatten_list_of_lists(pks)
-            + ["xbrl_factoid"]
-            + ["table_name"]
-        )
+        pks = [
+            "table_name",
+            "xbrl_factoid",
+        ] + pudl.helpers.dedupe_n_flatten_list_of_lists(pks)
```

Comment:
i put these two first bc it looks prettier in the dfs when you `set_index(self.exploded_pks)` 🎨